client/src/scene/SceneGameArena.ts

```diff
@@ -184,11 +197,47 @@ export class SceneGameArena extends Phaser.Scene {
             moved = this.gameState.moveIfCan(
                 Tetromino.rotateCW // clock wise
             );
+        } else if (this.keys.shift.isDown && this.gameState.playerId != null) {
```

Comment:
I would maybe add the `!this.gameState.isInOppositeSection()` here too, though that's an assumption about the game behavior I'm making

Reply:
Yeah not sure whether that's what we want. Otherwise, would already requested trades no longer be valid in opposite sections as well?